deps.macos/90-libdatachannel.zsh

```diff
@@ -0,0 +1,68 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='libdatachannel'
+local version='0.18.3'
+local url='https://github.com/Sean-Der/libdatachannel.git'
+local hash='f16c96bdb5628c6ed6549769b53773942401a98f'
+
+## Build Steps
+setup() {
+  log_info "Setup (%F{3}${target}%f)"
+  setup_dep ${url} ${hash}
+}
+
+clean() {
+  cd "${dir}"
+
+  if [[ ${clean_build} -gt 0 && -d "build_${arch}" ]] {
+    log_info "Clean build directory (%F{3}${target}%f)"
+
+    rm -rf "build_${arch}"
+  }
+}
+
+config() {
+  autoload -Uz mkcd progress
+
+  log_info "Config (%F{3}${target}%f)"
+
+  args=(
+    ${cmake_flags}
+    -DUSE_MBEDTLS=1
+    -DNO_WEBSOCKET=1
+    -DNO_TESTS=1
+    -DNO_EXAMPLES=1
+    -DPYTHON_EXECUTABLE=$(brew --prefix)/opt/python3/bin/python3
+    -DPYTHON_LIBRARY=$(brew --prefix)/opt/python3/lib/
```

Comment:
You are building Mbeb TLS submodule inside of libdatachannel and not using the "system" one.

```suggestion
    -DUSE_SYSTEM_MBEDTLS=ON
```